<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[openBrowser]]>
    </line>
    <line>
      <![CDATA[login]]>
    </line>
    <line>
      <![CDATA[logout]]>
    </line>
    <line>
      <![CDATA[login]]>
    </line>
    <line>
      <![CDATA[deletecustomer]]>
    </line>
    <line>
      <![CDATA[deletecustomer :has been passed]]>
    </line>
    <line>
      <![CDATA[logout]]>
    </line>
    <line>
      <![CDATA[modifycustomer :has been skipped]]>
    </line>
    <line>
      <![CDATA[closeBrowser]]>
    </line>
  </reporter-output>
  <suite started-at="2024-12-04T16:16:09 IST" name="Default suite" finished-at="2024-12-04T16:16:21 IST" duration-ms="11368">
    <groups>
    </groups>
    <test started-at="2024-12-04T16:16:09 IST" name="Default test" finished-at="2024-12-04T16:16:21 IST" duration-ms="11368">
      <class name="com.actitime.generic.CustomerModule">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:09 IST" name="openBrowser" finished-at="2024-12-04T16:16:16 IST" duration-ms="7164" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[openBrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:17 IST" name="login" finished-at="2024-12-04T16:16:17 IST" duration-ms="388" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="createcustomer()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:17 IST" name="createcustomer" finished-at="2024-12-04T16:16:17 IST" duration-ms="41" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.actitime.generic.CustomerModule.createcustomer(CustomerModule.java:18)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createcustomer -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:18 IST" name="logout" finished-at="2024-12-04T16:16:18 IST" duration-ms="376" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logout]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:18 IST" name="login" finished-at="2024-12-04T16:16:19 IST" duration-ms="261" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="deletecustomer()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:19 IST" name="deletecustomer" finished-at="2024-12-04T16:16:19 IST" duration-ms="4" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[deletecustomer]]>
            </line>
            <line>
              <![CDATA[deletecustomer :has been passed]]>
            </line>
          </reporter-output>
        </test-method> <!-- deletecustomer -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:19 IST" name="logout" finished-at="2024-12-04T16:16:20 IST" duration-ms="1544" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logout]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method signature="modifycustomer()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:20 IST" name="modifycustomer" finished-at="2024-12-04T16:16:20 IST" depends-on-methods="com.actitime.generic.CustomerModule.createcustomer" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CustomerModule.modifycustomer()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CustomerModule.modifycustomer()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
            <line>
              <![CDATA[modifycustomer :has been skipped]]>
            </line>
          </reporter-output>
        </test-method> <!-- modifycustomer -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.actitime.generic.CustomerModule@a38c7fe]" started-at="2024-12-04T16:16:20 IST" name="closeBrowser" finished-at="2024-12-04T16:16:21 IST" duration-ms="422" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[closeBrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.actitime.generic.CustomerModule -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
